typeOf1.js

```diff
@@ -0,0 +1,29 @@
+var x, y;

```

Comment:
Try to open PR (pull request) like this one, so I could better review your work.